Isort brings in its own copy of tomli rather than using the system one.

Not suitable for upstream.

--- isort-5.13.2/setup.py
+++ isort-5.13.2/setup.py
@@ -2,11 +2,15 @@
 from setuptools import setup
 
 packages = \
-['isort', 'isort._vendored.tomli', 'isort.deprecated', 'isort.stdlibs']
+['isort', 'isort.deprecated', 'isort.stdlibs']
 
 package_data = \
 {'': ['*']}
 
+install_requires = [
+    "tomli;python_version<'3.11'",
+]
+
 extras_require = \
 {'colors': ['colorama>=0.4.6']}
 
@@ -29,6 +33,7 @@ setup_kwargs = {
     'url': 'https://pycqa.github.io/isort/',
     'packages': packages,
     'package_data': package_data,
+    'install_requires': install_requires,
     'extras_require': extras_require,
     'entry_points': entry_points,
     'python_requires': '>=3.8.0',
--- isort-5.13.2/isort/settings.py
+++ isort-5.13.2/isort/settings.py
@@ -50,7 +50,7 @@ else:
     if sys.version_info >= (3, 11):
         import tomllib
     else:
-        from ._vendored import tomli as tomllib
+        import tomli as tomllib
 
 _SHEBANG_RE = re.compile(rb"^#!.*\bpython[23w]?\b")
 CYTHON_EXTENSIONS = frozenset({"pyx", "pxd"})
